doc/classes/StaticBody2D.xml

```diff
@@ -7,7 +7,7 @@
 		Static body for 2D physics.
 		A static body is a simple body that doesn't move under physics simulation, i.e. it can't be moved by external forces or contacts but its transformation can still be updated manually by the user. It is ideal for implementing objects in the environment, such as walls or platforms. In contrast to [RigidBody2D], it doesn't consume any CPU resources as long as they don't move.
 		They have extra functionalities to move and affect other bodies:
-		[b]Static transform change:[/b] Static bodies can be moved by animation or script. In this case, they are just teleported and don't affect other bodies on their path.
+		[b]Static transform change:[/b] Static bodies [i]can[/i] be moved by animation or script. In this case, they are just teleported and don't affect other bodies on their path. Use [AnimatableBody2D] instead of [StaticBody2D] if you need a moving static body that affects other bodies on its path.
```

Comment:
WDYT about somehow mentioning the "not affecting other bodies" part inside `<brief_description>` as well? IMO current phrasing implies that StaticBodies can be used to create programatically moving walls or platforms while in reality they shouldn't. The same goes for StaticBody3D